### ISSUE TITLE
[Feature Request] Smart quotes

### ISSUE THREAD

**Issue by icyphox** [lainsce/quilter#195]
Since Quilter focuses on writing, I thought it'd be great if we had a feature to replace normal quotes (`'`/`"`) with typographical ones (`‘’`/`“”`). Maybe in the form of a toggle.
What are your thoughts?

**Comment by icyphox** [comment]
Yeah, it does in preview mode. But wouldn’t it make sense to have it _actually_ use smart quotes while editing?

**Comment by lainsce** [comment]
The Edit View doesn't use fancy stuff like smart quotes because you're just writing, and writing, and writing away a story, it doesn't need fancy stuff other than focus-related things.

**Comment by icyphox** [comment]
Sure, that’s fair. But I generally export the document outside of Quilter. I don’t use the built-in export feature, which means I don’t get the smart quotes :/

**Comment by lainsce** [comment]
Maybe set your outside-exporter to read the quotes and make them curly?

**Comment by icyphox** [comment]
Haha, I’m not sure that’s possible. But alright :)
